src/collects/seashell/backend/dispatch.rkt

```diff
@@ -108,10 +112,21 @@
             (send-message connection `#hash((id . -4) (success . #t)
                                             (result . #hash((pid . ,pid) (test_name . ,test-name) (result . ,test-res)))))]
            [(list pid test-name "error" exit-code stderr)
+            ;; check for asan error message in "{project}/{question}/.asan.{pid}"
```

Comment:
Pass the error message in the list.  (Note: you'll have to use serialize,
look at the compiler code to see how that works)
On Apr 15, 2016 10:21 AM, "Yu Li" <notifications@github.com> wrote:

> In src/collects/seashell/backend/dispatch.rkt
> <https://github.com/cs136/seashell/pull/512#discussion_r59881257>:
>
> > @@ -108,10 +112,21 @@
> >              (send-message connection `#hash((id . -4) (success . #t)
> >                                              (result . #hash((pid . ,pid) (test_name . ,test-name) (result . ,test-res)))))]
> >             [(list pid test-name "error" exit-code stderr)
> > +            ;; check for asan error message in "{project}/{question}/.asan.{pid}"
>
> Do you want the error message still in stderr if its invoked from
> runner.rkt? In dispatch.rkt I added a new filed for sanitizerMsg in the
> "type: done" api because I thought we could potentially separate it out
> from user fprintf(stderr ..) and we could do some highlight.
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/cs136/seashell/pull/512/files/a3c9f9e1d686bbd01503cd73212432f9a87b788f#r59881257>
>